glauth/defaults/main.yaml

```diff
@@ -0,0 +1,63 @@
+# vim:ts=2:sw=2:et:ai:sts=2
+---
+glauth__config_file: /etc/glauth/glauth.toml
+glauth__install_ferm_svc: false  # Not implemented yet.
+
+# General options
+glauth__enable_debug: false
+glauth__log_to_syslog: false
+glauth__structured_log: false
+
+# Services that can be enabled
+glauth__service_ldap:
+  enabled: false
+
+glauth__service_ldaps:
+  enabled: false
+
+glauth__service_api:
+  enabled: false
+
+# Other options
+glauth__config_tracing:
+  enabled: false
+
+glauth__config_behaviors:
+  LimitFailedBinds: false
+  IgnoreCapabilities: false
+
+# glauth__config_backends: List of enabled backends. Each of them should
+# have, at least, `datastore` and `baseDN`.
+glauth__config_backends: []
+
+# glauth__datastore_groups: List of groups, passed to the config backend
```

Comment:
I am not convinced by the `datastore` prefix here, as these are for the "config" datastore only. Maybe `glauth__config_groups` or `glauth__local_groups`?

Reply:
Hm, good point. Would you be okay with one of `glauth__config_file_groups` or `glauth__config_datastore_groups`?

Reply:
(I'm okay with `glauth__config_groups` too!)

Reply:
`glauth__config_datastore_groups` seems good to me